frontend/src/components/pages/admin/AssessmentEditorPage.tsx

```diff
@@ -105,18 +110,36 @@ const AssessmentEditorPage = (): React.ReactElement => {
     }
   };
 
+  const onAfterModify = (options?: { redirect?: boolean }) => {
+    // Mark the form as clean after updating.
+    resetForm(getValues());
```

Comment:
we only need to reset form if we aren't redirecting right? or am i misunderstanding